submissions/boluAkintoye/html/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Product List</title>
+</head>
+<body>
+    <nav>
+        <h2>Welcome to Aloe House of Bags</h2>
+        <a href="index.html">Our Products</a>
+        <ul>
+
+            <img src="images/bag.jpg" alt="Brown bag">
+            <h1>J.QMEI Brown Bag </h1> 
+            <p><strong>Price: </strong><em>N20,000</em></p>
+            <p>Brown corporate 40cm leather bag with a long strap</p>
+        </div>
+            <img src= "images/bag2.jpg">
+            <h1>Blue Hermes Birkin Bag </h1>
+            <p><strong>Price: </strong><em>N10,000</em></p>
+            <p>Hermes special order Horseshoe 35cm shiny blue electric bag</p>   
+        </div>
+        </ul>
```

Comment:
When you are using a "ul" tag, I expect to see a list in it it. Since you are not using a list, you dont need the ul tag here. You should remove it.